Assets/Scripts/SceneSwitcher.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class SceneSwitcher : MonoBehaviour
+{
+    FadeInOut fade;
+
+    void Start()
+    {
+        fade = FindObjectOfType<FadeInOut>();
```

Comment:
Good for the prototype but in final product, FindObjectOfType should be changed. Finding an object from scene is CPU heavy calculation and could stutter the game at start.